ci/pipelines/aws-vault-template.yml

```diff
@@ -0,0 +1,178 @@
+meta:
+  docker-creds: &docker-creds
+    username: concourse
+    password: ((concourse_artifactory_password))
+  s3-params: &s3-params
+    bucket: ci-pix4d-concourse-pipeline
+    region_name: eu-west-1
+    access_key_id: ((concourse_user_access_key))
+    secret_access_key: ((concourse_user_secret_key))
+  release-params: &release-params
+    ARTIFACTORY_URL: https://artifactory.ci.pix4d.com/artifactory
+    ARTIFACTORY_USER: concourse
+    ARTIFACTORY_PASSWORD: ((concourse_artifactory_password))
+    CODESIGNER_TOKEN: ((codesigner_token))
+  status-handlers: &status-handlers
+    on_success:
+      put: cogito
+      no_get: true
+      inputs: [aws-vault.git]
+      params: {state: success}
+    on_failure:
+      put: cogito
+      no_get: true
+      inputs: [aws-vault.git]
+      params: {state: failure}
+    on_error:
+      put: cogito
+      no_get: true
+      inputs: [aws-vault.git]
+      params: {state: error}
+    on_abort:
+      put: cogito
+      no_get: true
+      inputs: [aws-vault.git]
+      params: {state: abort}
+
+##########################################################################################
+
+resource_types:
+  - name: cogito
+    type: registry-image
+    check_every: 24h
+    source:
+      repository: pix4d/cogito
+      tag: latest
+
+##########################################################################################
+
+resources:
+  - name: aws-vault.git
+    type: git
+    icon: git
+    webhook_token: ((concourse_gh_webhook))
+    source:
+      uri: git@github.com:pix4d/aws-vault.git
+      private_key: ((github_ssh_key))
+      branch: ((branch))
+      fetch_tags: true
+
+  - name: cogito
+    type: cogito
+    check_every: never
+    source:
+      owner: pix4d
+      repo: aws-vault
+      access_token: ((github_repo_status_token))
+      gchat_webhook: ((gchat_hook))
+
+  - name: archive-linux.s3
+    type: s3
+    icon: dump-truck
+    check_every: never
+    source:
+      regexp: artifacts/((project))/((branch))/aws-vault-v(\d+\.\d+\.\d+(-.+)*)-linux-amd64.tar.gz
+      <<: *s3-params
+
+  - name: archive-darwin-amd64.s3
+    type: s3
+    icon: dump-truck
+    check_every: never
+    source:
+      regexp: artifacts/((project))/((branch))/aws-vault-v(\d+\.\d+\.\d+(-.+)*)-darwin-amd64.zip
+      <<: *s3-params
+
+  - name: archive-darwin-arm64.s3
+    type: s3
+    icon: dump-truck
+    check_every: never
+    source:
+      regexp: artifacts/((project))/((branch))/aws-vault-v(\d+\.\d+\.\d+(-.+)*)-darwin-arm64.zip
+      <<: *s3-params
+
+  - name: aws-vault-builder.docker
+    type: registry-image
+    check_every: 24h
+    source:
+      repository: docker.ci.pix4d.com/aws-vault-builder
+      tag: add-aws-vault-builder
+      <<: *docker-creds
+
+##########################################################################################
+
+jobs:
+  - name: test
+    max_in_flight: 1
+    <<: *status-handlers
```

Comment:
Call it a base job and move max_in_flight there to avoid duplication.

Reply:
done